### PR TITLE
Update coding guidelines to use =>

### PR DESCRIPTION
**Pull request by Troy-Ferrell**
## Overview
Minor change to add =>

## Changes
- Fixes: # .

### CODE REVIEW

**Comment by wiwei** [comment]
Is there also some guidance we can have for multi-line getters/setters? i.e. this shows what we should do if we have a really small getter/setter, but leaves ambiguity if there is more complexity of getter/setters

File: Documentation/Contributing/CodingGuidelines.md [hunk 1]
```diff
@@ -171,7 +171,7 @@ Always declare an access modifier for all fields, properties and methods.
 protected int myVariable = 0;
 
 // property should have protected setter
-public int MyVariable { get { return myVariable; } }
+public int MyVariable { get => myVariable; }
```
Comment (keveleigh) [inline Documentation/Contributing/CodingGuidelines.md]: For this one could we even go one step further, or would that muddle the guideline?

```suggestion
public int MyVariable => myVariable;
```

**Comment by Troy-Ferrell** [comment]
> Is there also some guidance we can have for multi-line getters/setters? i.e. this shows what we should do if we have a really small getter/setter, but leaves ambiguity if there is more complexity of getter/setters

@wiwei done

**Pull request merged**